specification/alertsmanagement/resource-manager/readme.md

```diff
@@ -45,6 +45,15 @@ tag: package-preview-2021-08
 ```
 
 
+
+### Tag: package-2021-08
+
+These settings apply only when `--tag=package-2021-08` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-08'
+input-file:
+  - Microsoft.AlertsManagement/stable/2021-08-08/AlertProcessingRules.json
```

Comment:
Compared with other tags, `package-2021-08` only contain one file. Once other tags were ever released, the possible release of `package-2021-08` in the future will bring breaking change.